server/auth/store_test.go

```diff
@@ -300,6 +300,28 @@ func TestUserGrant(t *testing.T) {
 	}
 }
 
+func TestGetRootRole(t *testing.T) {
+	as, tearDown := setupAuthStore(t)
+	defer tearDown(t)
+
+	perm := &authpb.Permission{
+		PermType: authpb.READWRITE,
+		Key:      allKeys,
+		RangeEnd: []byte{0},
+	}
+
+	//get root role
+	r, err := as.RoleGet(&pb.AuthRoleGetRequest{Role: "root"})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	//check the role is root
+	if !reflect.DeepEqual(perm, r.Perm[0]) {
```

Comment:
Please prefer: 

```
assert.Equal(t, perm, ...)
```

Reply:
Hi, for code unifying, see line_470, line_547, line_562